package.json

```diff
@@ -4,14 +4,14 @@
   "description": "A set of primitives to build simple, flexible, WAI-ARIA compliant React autocomplete components",
   "main": "dist/downshift.cjs.js",
   "jsnext:main": "dist/downshift.esm.js",
-  "react-native": "native/index.js",
+  "react-native": "dist/downshift.native.cjs.js",
   "module": "dist/downshift.esm.js",
   "typings": "typings/index.d.ts",
   "scripts": {
     "add-contributor": "kcd-scripts contributors add",
     "build": "npm run build:web --silent && npm run build:native --silent",
     "build:web": "kcd-scripts build --bundle --p-react",
-    "build:native": "BUILD_REACT_NATIVE=true BUILD_FILENAME_PREFIX=react-native kcd-scripts build --out-dir native",
+    "build:native": "BUILD_REACT_NATIVE=true BUILD_FILENAME_SUFFIX=.native kcd-scripts build --bundle cjs --no-clean",
```

Comment:
What's the rationale around including the `--no-clean` flag here?

Reply:
`build:web` are and `build:native` are ran sequentially, `kcd-scripts build --bundle` removes the target directory, so its removed during `build:web` and I dont want to remove what was just built by it by running `build:native`

the goal for me was to not introduce a separate directory and Ive wanted to output react-native version into `dist`

Reply:
What do you think of adding `--no-clean` only to the `build` command, rather than `build:native`? That way, the `build:native` command can be run safely, clearing out the `dist` directory, while the `build` command would be able to handle the sequential case like you mentioned?

Reply:
that makes sense

Reply:
@eliperkins changed